code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -17,15 +17,14 @@
 	new /obj/item/radio/headset/heads/captain/alt(src)
 	new /obj/item/radio/headset/heads/captain(src)
 	new /obj/item/storage/belt/sabre(src)
-	new /obj/item/storage/box/gunset/pdh(src) // SKYRAT EDIT CHANGE - ORIGINAL: new /obj/item/gun/energy/e_gun(src)
+	new /obj/item/gun/energy/e_gun(src)
```

Comment:
I liked the Captain getting a sidearm instead of just ye old boring energy gun.

Reply:
The captain has a self recharging unique to literally only them laser in their office. I don't think they need another unique weapon alongside their super sabre and whatnot.

Reply:
Not to mention the Captain's gun was stupid OP.